fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +222,40 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+ReferenceReg=`opts_GetOpt1 "--refreg" $@`                            # In the cases when BOLD image is registered to a specified BOLD reference, this option 
```

Comment:
From the way the code is set up, it appears that the only thing that is ever checked is whether its value is 'nonlinear'.  i.e., the default of 'linear' doesn't trigger anything.  Given that, I'm wondering if the name of this option shouldn't be `--refregnonlinear` with values of either TRUE or FALSE (default).

Reply:
That messes up OneStepResampling, when NONE is also an option if no other bold is used for reference.